templates/lune/product-blanc.amp.html

```diff
@@ -17,11 +17,9 @@
 <html ⚡>
   {{> ../../utils/amp-page-head.snip.html}}
   <body class="lune lune-blanc lune-theme-light">
-
-    {{> partials/product/product.snip.html }}
-
     {{> ../../components/navbar/navbar.snip.html }}
 
+    {{> partials/product/product.snip.html }}
```

Comment:
Changed DOM order between header+sidebar and main content. It fixes a couple of bugs, and is more consistent with the rest of the pages